pgrx-sql-entity-graph/src/aggregate/mod.rs

```diff
@@ -299,10 +299,12 @@ impl PgAggregate {
                 #[allow(non_snake_case, clippy::too_many_arguments)]
                 #pg_extern_attr
                 fn #fn_name(this: #type_state_without_self, #(#args_with_names),*, fcinfo: ::pgrx::pg_sys::FunctionCallInfo) -> #type_state_without_self {
-                    <#target_path as ::pgrx::aggregate::Aggregate>::in_memory_context(
-                        fcinfo,
-                        move |_context| <#target_path as ::pgrx::aggregate::Aggregate>::state(this, (#(#arg_names),*), fcinfo)
-                    )
+                    unsafe {
+                        <#target_path as ::pgrx::aggregate::Aggregate>::in_memory_context(
+                            fcinfo,
+                            move |_context| <#target_path as ::pgrx::aggregate::Aggregate>::state(this, (#(#arg_names),*), fcinfo)
+                        )
+                    }
```

Comment:
I honestly am not entirely sure if this is actually correct to assert, either, but I **think** this fn in practice can only get called by Postgres. ^^;